server/app/services/question/ActiveAndDraftQuestions.java

```diff
@@ -112,38 +123,26 @@ private ActiveAndDraftQuestions(Version active, Version draft, Version withDraft
    * Inspects the provided version and returns a map who's key is the question name and value is a
    * set of programs that reference the given question in this version.
    */
-  private static ImmutableMap<String, ImmutableSet<ProgramDefinition>> buildReferencingProgramsMap(
+  private static ImmutableMap<Long, ImmutableSet<ProgramDefinition>> buildReferencingProgramsMap(
       Version version) {
-    // Different versions of a question can have distinct IDs while still
```

Comment:
Can you help me better understand why we moved away from this model (e.g. was it giving incorrect results / counts)? I'm a bit wary since I'm not 100% sure when we would associate the new draft question with all the programs it's used in.

Reply:
Originally I moved away from that model because I thought we need different counts for draft and active versions. Also mock had different counts. But now, after our discussion, I think it makes more sense to have the same count. Because conceptually whole question is assigned to the program. And you can't assign just draft or just active to a program. So not sure what exactly to do here. Maybe for now just keep same count and revert ActiveAndDraftQuestions changes?

Reply:
Per @jeanniefu 's feedback, I think we can address the "referencing programs" counts issue separately.

Given that, if it's not too much work, I'd vote for keeping the same count right now.

Reply:
Reverted back to using only questionName instead of ids.